diff --git a/pam.c b/pam.c
index c5a3001..392921d 100644
--- a/pam.c
+++ b/pam.c
@@ -276,7 +276,7 @@ pamauth(const char *user, const char *myname, int interactive, int nopass, int p
 		if (gethostname(host, sizeof(host)))
 			snprintf(host, sizeof(host), "?");
 		snprintf(doas_prompt, sizeof(doas_prompt),
-		    "\rdoas (%.32s@%.32s) password: ", myname, host);
+				"\r\033[97m[doas]\033[37m password for %.32s:\033[0m ", myname);
 
 		/* authenticate */
 		ret = pam_authenticate(pamh, 0);
-- 
2.24.1

